modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/FilterOrderControls.js

```diff
@@ -76,19 +119,109 @@ const FilterOrderControls = ({
 				</ClayManagementToolbar.Item>
 			)}
 
-			{sortingURL && (
+			{showDesignImprovements &&
+				orderDropdownItems &&
+				orderDropdownItems.length > 1 && (
```

Comment:
If we initialize `orderDropdownItems` to `[]` we don't need to check if exists before checking the length (and probably we won't need as much conditions in the previous comment)

Reply:
![Galaxy brain](https://media3.giphy.com/media/V0IdVIIW1y5d6/giphy.gif)